docker_images/k2_sherpa/tests/test_api.py

```diff
@@ -0,0 +1,40 @@
+import os
```

Comment:
By the way, can we delete the following files?

- docker_images/k2_sherpa/tests/samples/plane2.jpg
- docker_images/k2_sherpa/tests/samples/plane2.jpg

I think this PR does not use images.

Reply:
Sure you can !

The copy command just adds them for conveniency (avoid to add a lot of `if` within the docs to add a new lib :))

Reply:
Hi, I'm from a different time zone, I'm addressing now